js/tiles.js

```diff
@@ -314,6 +314,13 @@ const door = {
       this.wallRight = this.createImage("tiles/rooms/wall/dark.svg");
       this.ground = this.createImage("tiles/animations/glow.svg"); 
     },
+		 visit: function() {
```

Comment:
Properly intend it pls

Reply:
@abishekvashok check now, should be ok and if not - let me know

Reply:
Resolved.